### PR TITLE
Updated all licences and linters in this repo.

### PR DESCRIPTION
**Pull request by BenStarmerSmith**
## Proposed changes

Updated all licences and linters in this repo.

## Checklist

Before posting a PR ensure that from each of the below categories **AT LEAST ONE BOX HAS BEEN CHECKED**. If more than one category is applicable then more can be checked. Also ensure that the proposed changes have been filled out with relevant information for reviewers.

## Tests

- [ ] No tests required to be added. (For small changes that will be tested by CI/CD infrastructure).
- [ ] Added/Modified automated and PhantomHand CI tests (if a new class is added (Python or C++), the interface of that class must be unit tested).
- [x] Manually tested in simulation (if simulation specific or no hardware required to test the functionality). 
- [x] Manually tested on hardware (if hardware specific or related).

## Documentation

- [x] No documentation required to be added.
- [ ] Added documentation (For any new feature, explain what it does and how to use it. Write the documentation in a relevant space, e.g. Github, Confluence, etc).
- [ ] Updated documentation (For changes to pre-existing features mentioned in the documentation).

### CODE REVIEW

File: sr_demos/scripts/sr_demos/burn_in.py [hunk 1]
```diff
@@ -159,30 +166,30 @@ def execute_command_check(hand_commander, joint_states_config, joint_states,
...
+    if joint_prefix_name == 'rh_':
         hand_name = 'right_hand'
-    elif 'lh_' == joint_prefix:
+    elif joint_prefix_name == 'lh_':
         hand_name = 'left_hand'
     else:
         hand_name = 'two_hands'
 
-    hand_commander = SrHandCommander(name=hand_name)
+    hand_commander_ptr = SrHandCommander(name=hand_name)
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/burn_in.py]: why did you change the name as if the variable was a pointer?

File: sr_demos/scripts/sr_demos/burn_in.py [hunk 2]
```diff
@@ -159,30 +166,30 @@ def execute_command_check(hand_commander, joint_states_config, joint_states,
...
 
     # Get joint states for burn in demo
     burn_in_config_filename = '/home/user/projects/shadow_robot/base/src/'\
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/burn_in.py:185]: Since we use ROS can't we access this using rospkg so that it becomes less hardcoded and therefore more maintainable?

File: sr_demos/scripts/sr_demos/close.py [hunk 3]
```diff
@@ -53,20 +60,20 @@ def execute_trajectory(hand_commander, joint_states_no_id, joint_prefix, msg, ti
...
+    if joint_prefix_name == 'rh_':
         hand_name = 'right_hand'
-    elif 'lh_' == joint_prefix:
+    elif joint_prefix_name == 'lh_':
         hand_name = 'left_hand'
     else:
         hand_name = 'two_hands'
 
-    hand_commander = SrHandCommander(name=hand_name)
+    hand_commander_ptr = SrHandCommander(name=hand_name)
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/close.py]: SAME QUESTION AS BEFORE

File: sr_demos/scripts/sr_demos/demo.py [hunk 4]
```diff
@@ -62,7 +71,7 @@ def zero_tactile_sensors(self):
 
             # Collect SAMPLES_TO_COLLECT next samples and average them to filter out possible noise
             accumulator = []
-            for i in range(0, SAMPLES_TO_COLLECT):
+            for _ in range(0, SAMPLES_TO_COLLECT):
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/demo.py]: You don't need to have the 0 in the `range`

File: sr_demos/scripts/sr_demos/demo.py [hunk 5]
```diff
@@ -103,23 +112,24 @@ def confirm_touched(self):
         return touched
 
 
-class KeyboardPressDetector(object):
+class KeyboardPressDetector():
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/demo.py]: If you remove object, you can also remove the brackets :)

File: sr_demos/scripts/sr_demos/demo.py [hunk 6]
```diff
@@ -393,7 +400,7 @@ def sequence_lf(hand_commander, joint_states_config, tactile_reading):
     rospy.sleep(0.5)
     # Initialize wake time
     wake_time = time.time()
-    CONST_TIME_TO_COMPLETE_DEMO = 15
+    const_time_to_complete_demo = 15
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/demo.py]: This doesn't seem to be ever modified, so why reformating from upper case to lower case?

File: sr_demos/scripts/sr_demos/demo.py [hunk 7]
```diff
@@ -507,41 +513,42 @@ def execute_command_check(hand_commander, joint_states_config, joint_states,
...
+    if joint_prefix_name == 'rh_':
         hand_name = "right_hand"
-    elif 'lh_' == joint_prefix:
+    elif joint_prefix_name == 'lh_':
         hand_name = "left_hand"
     else:
         hand_name = "two_hands"
 
-    hand_commander = SrHandCommander(name=hand_name, prefix=joint_prefix)
+    hand_commander_ptr = SrHandCommander(name=hand_name, prefix=joint_prefix_name)
```
Comment (bdenoun) [inline sr_demos/scripts/sr_demos/demo.py]: same here about pointer

File: sr_example/scripts/sr_example/advanced/sr_latching_example.py [hunk 8]
```diff
@@ -23,13 +34,12 @@
...
 
-from __future__ import absolute_import
-import rospy
 import time
+import rospy
 from std_msgs.msg import Float64
 
 
-class LatchingExample(object):
+class LatchingExample():
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/advanced/sr_latching_example.py]: You can remove the brackets as well

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py [hunk 9]
```diff
@@ -86,8 +96,8 @@
     (position, orientation) = tf_listener.lookupTransform('ra_base', 'ra_flange',
                                                           tf_listener.getLatestCommonTime('ra_base',
                                                                                           'ra_flange'))
-except Exception as e:
-    raise ValueError(str(e))
+except Exception as exception:
+    raise ValueError(str(exception)) from exception
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: Use f-strings please

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_waypoints.py [hunk 10]
```diff
@@ -84,8 +93,8 @@
     (position, orientation) = tf_listener.lookupTransform('ra_base', 'ra_flange',
                                                           tf_listener.getLatestCommonTime('ra_base',
                                                                                           'ra_flange'))
-except Exception as e:
-    raise ValueError(str(e))
+except Exception as exception:
+    raise ValueError(str(exception)) from exception
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_waypoints.py]: use f-strings please

File: sr_example/scripts/sr_example/hand_examples/sr_right_hand_predefined_gestures.py [hunk 11]
```diff
@@ -1,31 +1,43 @@
...
 import termios
 import sys
 import tty
 import yaml
+import rospy
 from sr_robot_commander.sr_hand_commander import SrHandCommander
 
 
-class GraspExecution(object):
+class GraspExecution():
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/hand_examples/sr_right_hand_predefined_gestures.py]: can remove the brackets

File: sr_example/scripts/sr_example/hand_examples/sr_use_exported_states.py [hunk 12]
```diff
@@ -1,42 +1,51 @@
...
+# warehouse_states = {
+#   'state_1': {
+#      'joint_0': 0.00,
+#      'joint_1': 0.00
+#   },
+#   'state_2': {
+#      'joint_0': 0.00,
+#      'joint_1': 0.00
+#   }
+# }
```
Comment (bdenoun) [inline sr_example/scripts/sr_example/hand_examples/sr_use_exported_states.py:48]: Do we want to keep this commented out?

File: sr_grasp/scripts/grasp [hunk 13]
```diff
@@ -30,7 +36,7 @@ from sr_robot_msgs.msg import ShadowPST, ShadowContactStateStamped, GraspAction
 from gazebo_msgs.msg import ContactsState
 
 
-class InterpolatedJoint(object):
+class InterpolatedJoint():
```
Comment (bdenoun) [inline sr_grasp/scripts/grasp]: remove brackets

File: sr_grasp/scripts/grasp [hunk 14]
```diff
@@ -57,7 +63,7 @@ class InterpolatedJoint(object):
         return False
 
 
-class GraspNode(object):
+class GraspNode():
```
Comment (bdenoun) [inline sr_grasp/scripts/grasp]: can remove brackets

File: sr_grasp/scripts/grasp [hunk 15]
```diff
@@ -136,19 +143,18 @@ class GraspNode(object):
...
                     'tactile', ShadowPST, self.pst_tactile_cb)
             else:
-                rospy.logerr("Don't know how to handle %s tactiles!" %
-                             (self.hand.get_tactile_type()))
+                rospy.logerr(f"Don't know how to handle {self.hand.get_tactile_type()} tactiles!")
 
         rospy.loginfo("Ready to grasp")
 
-    def config_cb(self, config, level):
+    def config_cb(self, config, _level):
```
Comment (bdenoun) [inline sr_grasp/scripts/grasp]: why using a `_` at the beginning of the variable used in your signature? If you don't want to use `level`, AFAIK the convention is `level_`

File: sr_grasp/scripts/grasp [hunk 16]
```diff
@@ -210,7 +216,8 @@ class GraspNode(object):
         rospy.loginfo("Grasp preempted, stopping")
         self.grasp_server.set_preempted()
 
-    def joint_trajectory_to_position(self, traj):
+    @staticmethod
+    def joint_trajectory_to_position(traj):
```
Comment (bdenoun) [inline sr_grasp/scripts/grasp]: traj --> trajectory?

File: sr_grasp/scripts/quick_grasp [hunk 17]
```diff
@@ -1,38 +1,40 @@
...
 from actionlib import SimpleActionClient, GoalStatus
 from sr_robot_msgs.msg import GraspAction, GraspGoal
 from sr_hand.shadowhand_ros import ShadowHand_ROS
-from sr_grasp.utils import mk_grasp
 from sr_grasp import Grasp, GraspStash
-from builtins import input
 
 
-class QuickGraspNode(object):
+class QuickGraspNode():
```
Comment (bdenoun) [inline sr_grasp/scripts/quick_grasp]: can remove brackets

File: sr_grasp/scripts/quick_grasp [hunk 18]
```diff
@@ -84,8 +85,8 @@ if __name__ == '__main__':
         print("")
         print("Grasps:")
         for i, g in enumerate(node.stash.get_all()):
-            print(str(i) + " - " + g.id)
-        print("Current grasp: %s" % node.grasp.id)
+            print(f"{str(i)} - {g.id}")
```
Comment (bdenoun) [inline sr_grasp/scripts/quick_grasp]: If you use f-strings you don't need the str I guess

File: sr_grasp/src/sr_grasp/__init__.py [hunk 19]
```diff
@@ -104,7 +100,7 @@ def _set_posture_point(self, posture, positions, point=0):
         posture.points[point] = jtp
 
 
-class GraspStash(object):
+class GraspStash():
```
Comment (bdenoun) [inline sr_grasp/src/sr_grasp/__init__.py]: can remove brackets

File: sr_grasp/src/sr_grasp/__init__.py [hunk 20]
```diff
@@ -158,20 +154,21 @@ def as_yaml(self):
...
-            return False
-        else:
-            loginfo("Loaded grasps from file: %s" % (fname))
-            return True
+        with open(fname, "r") as stream:
+            try:
+                pdata = yaml.safe_load(stream)
+                self.load_yaml(pdata)
+            except Exception as exc:
+                logerr("Failed to load YAML grasp file: %s error:%s" % (fname, exc))
```
Comment (bdenoun) [inline sr_grasp/src/sr_grasp/__init__.py]: Use f-strings?

File: sr_grasp/src/sr_grasp/__init__.py [hunk 21]
```diff
@@ -158,20 +154,21 @@ def as_yaml(self):
...
+                return False
+            else:
+                loginfo("Loaded grasps from file: %s" % (fname))
```
Comment (bdenoun) [inline sr_grasp/src/sr_grasp/__init__.py]: Use f-strings?

**Review by bdenoun — CHANGES_REQUESTED** [review]
A bit confused about why some variables have been appended with a `_ptr`

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: I think thats outside the scope of this linter change. You could add it to the tech debt as I think this fits it more, as it would likely be good to have a story to make this change across the repo and test it thouroghly

File: sr_demos/scripts/sr_demos/burn_in.py [hunk 22]
```diff
@@ -1,32 +1,39 @@
...
+import rospy
 from sr_robot_commander.sr_hand_commander import SrHandCommander
-from sr_utilities.hand_finder import HandFinder
 
 
 def burn_in_demo(hand_commander, burn_in_config):
-    for x in range(0, 100):
-        rospy.loginfo("We're on iteration number %d" % (x))
+    for iteration in range(100):
+        rospy.loginfo("We're on iteration number %d" % iteration)
```
Comment (maxzieba) [inline sr_demos/scripts/sr_demos/burn_in.py]: f-string

File: sr_demos/scripts/sr_demos/close.py [hunk 23]
```diff
@@ -53,20 +60,20 @@ def execute_trajectory(hand_commander, joint_states_no_id, joint_prefix, msg, ti
...
+    if joint_prefix_name == 'rh_':
         hand_name = 'right_hand'
-    elif 'lh_' == joint_prefix:
+    elif joint_prefix_name == 'lh_':
         hand_name = 'left_hand'
     else:
         hand_name = 'two_hands'
 
-    hand_commander = SrHandCommander(name=hand_name)
+    hand_commander_class = SrHandCommander(name=hand_name)
```
Comment (maxzieba) [inline sr_demos/scripts/sr_demos/close.py]: misleading variable, its an instance and not class - why the need to change it?

File: sr_demos/scripts/sr_demos/demo.py [hunk 24]
```diff
@@ -151,13 +161,17 @@ def sequence_th(hand_commander, joint_states_config):
...
 def sequence_ff(hand_commander, joint_states_config):
     rospy.loginfo("FF demo started")
 
     rospy.sleep(1)
+    sequence_ff_commands(hand_commander, joint_states_config)
+
+    rospy.loginfo("FF demo completed")
+
+
+def sequence_ff_commands(hand_commander, joint_states_config):
```
Comment (maxzieba) [inline sr_demos/scripts/sr_demos/demo.py:175]: why the method name change?

File: sr_demos/scripts/sr_demos/demo.py [hunk 25]
```diff
@@ -231,10 +245,6 @@ def sequence_ff(hand_commander, joint_states_config):
     execute_command_check(hand_commander, joint_states_config, 'zero_wr', 0.4, 0.4)
     execute_command_check(hand_commander, joint_states_config, 'start_pos', 1.5, 1.5)
 
-    rospy.loginfo("FF demo completed")
-
-    return
```
Comment (maxzieba) [inline sr_demos/scripts/sr_demos/demo.py:236]: why removing the loginfo?

File: sr_demos/scripts/sr_demos/demo.py [hunk 26]
```diff
@@ -552,42 +559,42 @@ def execute_command_check(hand_commander, joint_states_config, joint_states,
...
-                                tactile_reading, args.hand_type)
+    kpd = KeyboardPressDetector(hand_commander_class, demo_states_with_prefix,
+                                tactile_reading_class, args.hand_type)
     keyboard_thread = Thread(target=kpd.run)
     keyboard_thread.start()
 
     while not rospy.is_shutdown():
         # Check the state of the tactile sensors
-        touched = None
+        touched_finger = None
```
Comment (maxzieba) [inline sr_demos/scripts/sr_demos/demo.py:569]: why the rename?

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py (hunk 9 above)
Comment (maxzieba) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: isn't ValueError available by default? No need to import from exception

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (bdenoun) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: Why not keep `hand_commander`?

File: sr_example/scripts/sr_example/hand_examples/sr_right_hand_partial_traj_advanced.py [hunk 27]
```diff
@@ -288,8 +304,8 @@ def construct_trajectory_point(posture, duration):
     joint_trajectory.header.stamp = start_time + rospy.Duration.from_sec(float(trajectory_start_time))
     joint_trajectory.joint_names = list(open_hand_current.keys())
     joint_trajectory.points = []
-    trajectory_point = construct_trajectory_point(open_hand_current, 1.0)
-    joint_trajectory.points.append(trajectory_point)
+    this_trajectory_point = construct_trajectory_point(open_hand_current, 1.0)
```
Comment (maxzieba) [inline sr_example/scripts/sr_example/hand_examples/sr_right_hand_partial_traj_advanced.py:307]: why rename? the 'this_' prefix doesnt feel right :(

File: sr_grasp/scripts/grasp [hunk 28]
```diff
@@ -95,10 +103,9 @@ class GraspNode(object):
         # as it blocks for a while detecting hands.
         rospy.loginfo("Detecting hand...")
         self.hand = ShadowHand_ROS()
-        t = "Unknown"
-        t = self.hand.hand_type
-        rospy.loginfo("Found %s hand with %s tactile." %
-                      (t, self.hand.get_tactile_type()))
+        hand_type = "Unknown"
+        hand_type = self.hand.hand_type
```
Comment (maxzieba) [inline sr_grasp/scripts/grasp:107]: does this make sense?

File: sr_grasp/scripts/grasp [hunk 29]
```diff
@@ -271,7 +278,7 @@ class GraspNode(object):
         # Run the iterpolated motion, testing for contact as we go.
         # IDEA: Run the motion on a thread and use the tactile callbacks to
         # trigger the stop.
-        r = rospy.Rate(self.interpolation_rate)
+        inter_rate = rospy.Rate(self.interpolation_rate)
```
Comment (maxzieba) [inline sr_grasp/scripts/grasp]: interpolation_rate

File: sr_grasp/scripts/quick_grasp [hunk 30]
```diff
@@ -70,8 +71,8 @@ class QuickGraspNode(object):
         grasp = Grasp()
         grasp.id = name
         positions = self.hand.read_all_current_positions()
-        for k, v in positions.items():
-            positions[k] = radians(v)
+        for key, value in positions.items():
+            positions[key] = radians(value)
```
Comment (maxzieba) [inline sr_grasp/scripts/quick_grasp]: I believe we should start using list comprehension style of doing this kind of things to be more pythonic and more efficient :)

File: sr_grasp/scripts/quick_grasp [hunk 31]
```diff
@@ -84,8 +85,8 @@ if __name__ == '__main__':
         print("")
         print("Grasps:")
         for i, g in enumerate(node.stash.get_all()):
```
Comment (maxzieba) [inline sr_grasp/scripts/quick_grasp]: i is ok, but rename g to grasp :)

File: sr_grasp/src/sr_grasp/__init__.py [hunk 32]
```diff
@@ -46,30 +45,26 @@ def __init__(self):
...
+        grasp.id = msg.id  # pylint: disable=C0103,W0201
+        grasp.pre_grasp_posture = deepcopy(msg.pre_grasp_posture)  # pylint: disable=W0201
+        grasp.grasp_posture = deepcopy(msg.grasp_posture)  # pylint: disable=W0201
+        grasp.grasp_pose = deepcopy(msg.grasp_pose)  # pylint: disable=W0201
+        grasp.grasp_quality = msg.grasp_quality  # pylint: disable=W0201
+        grasp.pre_grasp_approach = deepcopy(msg.pre_grasp_approach)  # pylint: disable=W0201
+        grasp.post_grasp_retreat = deepcopy(msg.post_grasp_retreat)  # pylint: disable=W0201
+        grasp.post_place_retreat = deepcopy(msg.post_place_retreat)  # pylint: disable=W0201
+        grasp.max_contact_force = msg.max_contact_force  # pylint: disable=W0201
+        grasp.allowed_touch_objects = deepcopy(msg.allowed_touch_objects)  # pylint: disable=W0201
```
Comment (maxzieba) [inline sr_grasp/src/sr_grasp/__init__.py:57]: bit weird as Grasp should have those variables in the constructor defined after inheriting from moveit_msgs...

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: Its used within a class earlier in the file and you can't have that according to the linter.

File: sr_demos/scripts/sr_demos/close.py (hunk 23 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/close.py]: Its used within a class earlier in the file and you can't have that according to the linter.

File: sr_demos/scripts/sr_demos/demo.py (hunk 24 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/demo.py:175]: This was a change by Beatriz to make it match the above function:
https://github.com/shadow-robot/sr_interface/blob/1542a45dbb9901bea085be880258e351f96cbcc7/sr_demos/scripts/sr_demos/demo.py#L156-L176

File: sr_demos/scripts/sr_demos/demo.py (hunk 25 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/demo.py:236]: It was move to the function that calls this function:
https://github.com/shadow-robot/sr_interface/blob/SP-207_fix_licence_and_lint/sr_demos/scripts/sr_demos/demo.py#L165-L171

File: sr_demos/scripts/sr_demos/demo.py (hunk 26 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/demo.py:569]: It's a duplicate variable name in a different scope which the linter doesn't like. (Same reason as the class instance vars)

File: sr_grasp/scripts/grasp (hunk 28 above)
Comment (BenStarmerSmith) [reply sr_grasp/scripts/grasp:107]: It was what was previously was used, could be something we should change as part of the tech debt

File: sr_grasp/src/sr_grasp/__init__.py (hunk 32 above)
Comment (BenStarmerSmith) [reply sr_grasp/src/sr_grasp/__init__.py:57]: Im not sure how this works as I don't work with msg's much, but we can have a talk with Beatriz about it tommorow as this aws one of her changes :)

File: sr_grasp/scripts/quick_grasp (hunk 30 above)
Comment (BenStarmerSmith) [reply sr_grasp/scripts/quick_grasp]: Can you leave me an example of what you mean please :) Do you mean have a new dictonary created with the updated value so we aren't editing a dictonary we are also iterating through?

File: sr_demos/scripts/sr_demos/demo.py (hunk 25 above)
Comment (maxzieba) [reply sr_demos/scripts/sr_demos/demo.py:236]: Not sure if wrapping all of the execute function into another function makes its better (style-wise), but ok :) 
]

File: sr_grasp/scripts/quick_grasp (hunk 30 above)
Comment (maxzieba) [reply sr_grasp/scripts/quick_grasp]: image

Comment (maxzieba) [reply sr_grasp/scripts/quick_grasp]: func:'ap1' args:[(), {}] took: 0.2770 sec
func:'ap2' args:[(), {}] took: 0.0758 sec

Comment (maxzieba) [reply sr_grasp/scripts/quick_grasp]: func:'ap1' args:[(), {}] took: 0.2770 sec
func:'ap2' args:[(), {}] took: 0.0758 sec

Comment (maxzieba) [reply sr_grasp/scripts/quick_grasp]: I know it's not much but general performance improvement is a good thing :)

File: sr_grasp/src/sr_grasp/__init__.py (hunk 32 above)
Comment (maxzieba) [reply sr_grasp/src/sr_grasp/__init__.py:57]: after a chat with @bdenoun we think it's mostly an pylinter issue, would be great to investigate this issue

Comment (BenStarmerSmith) [reply sr_grasp/src/sr_grasp/__init__.py:57]: I think it likely is the case, maybe it doesn't support msg's

File: sr_grasp/scripts/quick_grasp (hunk 30 above)
Comment (BenStarmerSmith) [reply sr_grasp/scripts/quick_grasp]: Changed :)

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (bdenoun) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: ahhhh, you mean it complains that this might be re-defined?

File: sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py (hunk 9 above)
Comment (BenStarmerSmith) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: https://pylint.pycqa.org/en/latest/user_guide/messages/warning/raise-missing-from.html

Comment (BenStarmerSmith) [reply sr_example/scripts/sr_example/hand_and_arm_examples/sr_right_hand_arm_ef_pos.py]: no its still needed

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (bdenoun) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: maybe `hand_commander_` would be less confusing?

File: sr_demos/scripts/sr_demos/demo.py (hunk 6 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/demo.py]: Constants cannot be initalised in functions according to PEP8

File: sr_demos/scripts/sr_demos/burn_in.py (hunk 2 above)
Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: Yeah exactly!

Comment (BenStarmerSmith) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: I think that would be more confusing as it'd make it look like a un-used variable, I think changing ptr to instance is much clearer

Comment (bdenoun) [reply sr_demos/scripts/sr_demos/burn_in.py:185]: all right

**Review by bdenoun — APPROVED** [review]

**Review by maxzieba — APPROVED** [review]
